bigbluebutton-html5/imports/ui/components/media/webcam-draggable-overlay/component.jsx

```diff
@@ -162,7 +162,7 @@ class WebcamDraggable extends Component {
 
   handleWebcamDragStop(e, position) {
     const { webcamDraggableDispatch, singleWebcam } = this.props;
-    const targetClassname = e.target.className;
+    const targetClassname = JSON.stringify(e.target.className);
```

Comment:
> `JSON.stringify(e.target.className)` 

What are the possible values for `e.target.className`?

Reply:
> Return Value: | A String, representing the class, or a space-separated list of classes, of an element